src/EnergyPlus/Construction.cc

```diff
@@ -1970,6 +1970,31 @@ bool ConstructionProps::isGlazingConstruction(EnergyPlusData &state) const
                              BITF(DataHeatBalance::MaterialGroup::WindowSimpleGlazing));
 }
 
+Real64 ConstructionProps::setThicknessPerpendicular(EnergyPlusData &state, Real64 userValue)
+{
+    // Limits set here are similar to limits used in HydronicSystemBaseData::sizeRadiantSystemTubeLength
+    // which is for autosizing tube length.  The limits are not as strictly enforced here because they are
+    // not being used for autosizing so more latitude with the values is desired.  Warnings will still alert
+    // the user if values seem outside of the ordinary range anticipated.
+    Real64 returnValue = userValue / 2.0; // Divide by two because internally the half distance will be used to calculate CTFs
+    if (returnValue <= 0.001) {           // lowest reasonable value for "half" the tube spacing in meters
+        ShowWarningError(state, "ConstructionProperty:InternalHeatSource has a tube spacing that is less than 2 mm.  This is not allowed.");
+        ShowContinueError(
+            state, "Construction=" + this->Name + " has this problem.  The tube spacing has been reset to 0.15m (~6 inches) for this construction.");
+        ShowContinueError(state, "As per the Input Output Reference, tube spacing is only used for 2-D solutions and autosizing.");
+        returnValue = 0.075;          // default "half" tube spacing in meters (roughly equivalent to 15cm or 6 inches of tube spacing)
+    } else if (returnValue < 0.005) { // below this value for "half" the tube spacing in meters throw a warning
```

Comment:
Very minor: Here it is slightly different from the document description about the two-end limits: the code will warn when the tube distance `smaller than` 0.01, which mean the tube is required to `greater or equal` than 0.01m; vs. the document's 'greater than'. 

The same thing for the other limit of 1m: code requires `less than or equal' to be warning-free; the doc says 'less than' 1m. So it is really about which category these two end values of 0.01m and 1m fall into.

Reply:
Agreed. There is a follow up issue for this so those can be addressed then. Too minor of an issue to waste more developer or CI time with this. I pulled develop and am running the unit tests locally now. This will merge soon.

Reply:
Thanks--got it on my list of things to clean-up post-release!